File: vueapp/src/models/IAuditable.js

function IAuditable (dateCreated, dateModified, creatorName, modifierName)
{
	this.DateCreated 	= dateCreated
	this.DateModified 	= dateModified
	this.CreatorName 	= creatorName
	this.ModifierName 	= modifierName
}

export { IAuditable }

File: coreLogic/Interfaces/IAuditable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace coreLogic.Interfaces;
public interface IAuditable
{
	DateTime DateCreated { get; set; }

	DateTime DateModified { get; set; }

	int CreatorId { get; set; }

	int ModifierId { get; set; }
}